### ISSUE TITLE
Lack of Timelock

### ISSUE THREAD

**Issue by GalloDaSballo** [code-423n4/rulebook#7]
From a contest am currently judging, this finding is used to justify a medium severity for "lack of timelock"
https://github.com/code-423n4/2021-11-overlay-findings/issues/120

**Comment by GalloDaSballo** [comment]
I don't believe there's a way to prove a system will use a timelock, unless the timelock mechanism is added to the code.

While I'm a strong proponent to keep Owner Privilege findings (owner can rug, owner can frontrun, owner can scam, etc..), I don't think a generic finding without a specific proof of concept should keep medium severity

**Comment by GalloDaSballo** [comment]
While some wardens raised the idea that you can tell that the protocol will use a timelock based on tests or testnet deployments, my experience tells me this is not the case.

Most protocols will have a deployer account (EOA) run all the deployments and settings, they will then review the settings and finally transfer ownership to the timelock.

This is common-sense, why would you wait X days to set parameters on first deploy?

As such, for any time a warden suggest to use a timelock, the sponsor can lie and say they do.

This to me means the finding is ultimately not helpful, it may be best to add it at the bottom of the disclosure "We assume the reviewed code will use TimeLock for sensitive operations"

**Comment by 0xean** [comment]
Thanks for the ping @GalloDaSballo - My feelings have certainly evolved from the judging of the swivel contest which was one of my first.  

I do believe the finding there to be valid, but would probably move it to a Low Severity at this point in my judging.  The events  for critical state changes not being emitted seems like it qualifies for that severity.

```
1 — Low: Low: Assets are not at risk. State handling, function incorrect as to spec, issues with comments.
```

I could see that some would argue it's a non-critical since its an event, but I think due to the power of the privilege addresses the low is more fitting
